packages/admin/admin-rte/src/core/translation/xml/transformStateToXml.ts

```diff
@@ -0,0 +1,171 @@
+import { CharacterMetadata, ContentBlock, ContentState } from "draft-js";
```

Comment:
This code was originally implemented in #1040. Please include the unit tests that were added.